lurn.gemspec

```diff
@@ -1,7 +1,5 @@
 # coding: utf-8
-lib = File.expand_path('../lib', __FILE__)
-$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'lurn/version'
```

Comment:
@Nakilon Your commit message mentions that using `require_relative` is a more current practice. Can you elaborate on that? I've done a little bit of googling and can't find anything on the topic.

Reply:
Because there were no `require_relative` and `require` acceoted only absolute path so you had to wither invoke `File.expand_path` for every `require` command you need or edit the `$LOAD_PATH` to affect all the following `require`s. `require_relative` made the pollution of global var unneccessary.

Reply:
Thanks @Nakilon. Merging now.